a-list-apart/a-list-out.csv

```diff
@@ -0,0 +1,488 @@
+title,author,tags,link,source
```

Comment:
EDIT: unless the intent is to have some boilerplate example code as part of your repo, in which case you can make a case for including some pre-made CSVs. Depends on what exactly you intend to do with this data.